docs/UserGuide.md

```diff
@@ -18,30 +21,6 @@ Chesse Inventory Management (CHIM) is a **desktop app for managing contacts, opt
 
 ## Features
 
-<div markdown="block" class="alert alert-info">
-
-**:information_source: Notes about the command format:**<br>
```

Comment:
Are we keeping this portion? Can refer below for the modified command in @weixue123 's repo

[code portion](https://github.com/AY2021S2-CS2103-W16-2/tp/compare/master...weixue123:inc/doc#diff-b50feaf9240709b6b02fb9584696b012c2a69feeba89e409952cc2f401f373fbR28)

Reply:
Yeap , will merge his changes into my branch after his PR has been merged